UI/forms/OBSBasic.ui

```diff
@@ -487,7 +487,7 @@
      <x>0</x>
      <y>0</y>
      <width>1079</width>
-     <height>21</height>
+     <height>23</height>
```

Comment:
This seems to be the only change that actually also changes a value? Is this intended?